docs/extensions/server/applications.rst

```diff
@@ -18,7 +18,71 @@ Manage Applications against Extra Cluster Managers
 
 .. versionadded:: 1.6.0
 
-.. caution:: unstable
+Application Operation
```

Comment:
keep a consistent style with authentication?

Reply:
changed to `Build A Custom Application Operation`

Reply:
I didn't mean only this one but the full hierarchy of this page

Reply:
oh, I got it now. updated